llvm/lib/Target/AIE/AIE2GenInstrInfo.td

```diff
@@ -94,6 +94,8 @@ let Itinerary = II_NE in {
 }
 let Itinerary = II_OR in {
   def OR : AIE2_alu_r_rr_inst_alu<0b0101, (outs eR:$mRx), (ins eR:$mRx0, eR:$mRy), "or", "$mRx, $mRx0, $mRy">;
+  let isCodeGenOnly = 1 in 
+    def MOV_OR : AIE2_alu_r_rr_inst_alu_mov_gpr<0b0101, (outs eR:$mRx), (ins eR:$mRx0), "or", "$mRx, $mRx0, $mRx0">;
```

Comment:
Mentioned offline: this is nit-picking but our ISA gives an alternate ASM syntax to "add immediate" instructions when the immediate is 0: `MOVE <adst>, <src>`. So I'd rather use `ADD_r_ri` instead of `MUL`

Reply:
Hi @gbossu am not sure what are you asking for ? and how MUL relates to what to this PR.

I think ADD immediate with 0 value being considered as MOVE and what we are doing here are two distinct things.

Reply:
Discussed offline : We used OR over ADD because ADD use carryBit which can result in incorrect result.